File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/CBuilder/ViewBandedDemo/ViewBandeDemoBands.cpp
//---------------------------------------------------------------------------

#include <vcl.h>
#pragma hdrstop

#include "ViewBandeDemoBands.h"
//---------------------------------------------------------------------------
#pragma package(smart_init)
#pragma link "cxButtons"
#pragma link "cxLookAndFeelPainters"
#pragma link "cxListBox"
#pragma link "cxContainer"
#pragma link "cxControls"
#pragma link "cxEdit"
#pragma link "cxGraphics"
#pragma link "cxLookAndFeels"
#pragma resource "*.dfm"
TViewBandeDemoBandsForm *ViewBandeDemoBandsForm;
//---------------------------------------------------------------------------
__fastcall TViewBandeDemoBandsForm::TViewBandeDemoBandsForm(TComponent* Owner)
  : TForm(Owner)
{
}
//---------------------------------------------------------------------------
void __fastcall TViewBandeDemoBandsForm::btnCancelClick(TObject *Sender)
{
	Close();
}
//---------------------------------------------------------------------------

File: Components/DevExpressVCL/Demos/ExpressQuantumGrid/Delphi/ViewBandedDemo/ViewBandeDemoBands.pas
unit ViewBandeDemoBands;

interface

uses
  Windows, Messages, Classes, Graphics, Controls, Forms,
  Dialogs, StdCtrls, cxButtons, cxListBox, cxLookAndFeelPainters,
  cxControls, cxContainer;

type
  TViewBandeDemoBandsForm = class(TForm)
    lbBands: TcxListBox;
    btnOK: TcxButton;
    btnCancel: TcxButton;
    procedure btnCancelClick(Sender: TObject);
  end;

implementation

{$R *.dfm}

procedure TViewBandeDemoBandsForm.btnCancelClick(Sender: TObject);
begin
  Close;
end;

end.
